app/controllers/index_urls_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class IndexURLsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  wrap_parameters false
+
+  attr_reader :index_urls
+
+  def start
+    begin
+      @job_id = params[:job_id]
+      @index_urls = params[:index_urls]
+      @job = enqueue_job
+    rescue BulkUrlUploader::Error => e
```

Comment:
Let's stop using this patter. It always ends just obfuscating what really happened. Let it fail loudly!